example-get-started-cv/generate.sh

```diff
@@ -0,0 +1,198 @@
+cp -r $HERE/code/.github .
+
+git add .
+tick
+git commit -m "Initialize Git repository"
+git tag -a "0-git-init" -m "Git initialized."
+
+dvc init
+tick
+git commit -m "Initialize DVC project"
+
+cp -r $HERE/code/data .
+git add data/.gitignore data/pool_data.dvc
+tick
+git commit -m "Add data"
+dvc pull
+
+# Remote active on this env only, for writing to HTTP redirect below.
+dvc remote add -d --local storage s3://dvc-public/remote/get-started-pools
+# Actual remote for generated project (read-only). Redirect of S3 bucket above.
+dvc remote add -d storage https://remote.dvc.org/get-started-pools
+git add .
+tick
+git commit -m "Configure default remote"
+dvc push
+
+
+cp -r $HERE/code/notebooks .
+pip install -r requirements.txt
+pip install jupyter
+jupyter nbconvert --execute 'notebooks/TrainSegModel.ipynb' --inplace
+git add .
+tick
+git commit -m "Add notebook using DVCLive"
+git tag -a "1-notebook-dvclive" -m "Notebook added."
+
+cp -r $HERE/code/src .
+cp $HERE/code/params.yaml .
+
+dvc stage add -n data_split \
+  -p base,data_split \
+  -d src/data_split.py -d data/pool_data \
+  -o data/train_data -o  data/test_data \
+  python src/data_split.py
+dvc repro
+git add .
+tick
+git commit -m "Create and run data split stage"
+dvc push
+
+dvc stage add -n train \
+  -p base,train \
+  -d src/train.py -d data/train_data \
+  -o models/model.pkl \
+  -M results/train/metrics.json \
+  --plots-no-cache results/train/plots \
+  python src/train.py
+dvc repro
+git add .
+tick
+git commit -m "Create and run train stage"
+dvc push
+
+dvc stage add -n evaluate \
+  -p base,evaluate \
+  -d src/evaluate.py -d models/model.pkl -d data/test_data \
+  -M results/evaluate/metrics.json \
+  --plots-no-cache results/evaluate/plots \
+  python src/evaluate.py
+dvc repro
+git add .
+tick
+git commit -m "Create and run evaluation stage"
+git tag -a "baseline-experiment" -m "Baseline experiment evaluation"
+dvc push
+
+export GIT_AUTHOR_NAME="David de la Iglesia"
+export GIT_AUTHOR_EMAIL="daviddelaiglesiacastro@gmail.com"
+export GIT_COMMITTER_NAME="$GIT_AUTHOR_NAME"
+export GIT_COMMITTER_EMAIL="$GIT_AUTHOR_EMAIL"
+
+git checkout -b "tune-architecture"
+
+unset GIT_AUTHOR_DATE
+unset GIT_COMMITTER_DATE
+
+dvc exp run --queue --set-param train.arch=alexnet
+dvc exp run --queue --set-param train.arch=resnet34
+dvc exp run --queue --set-param train.arch=vgg19_bn
+dvc exp run --queue --set-param train.arch=squeezenet1_1
+
+dvc exp run --run-all
+# Apply best experiment
+EXP=$(dvc exp show --csv --sort-by results/evaluate/metrics.json:dice_multi | tail -n 1 | cut -d , -f 1)
+dvc exp apply $EXP
+tick
+git commit -am "Run experiments tuning architecture"
+dvc push
+
+git checkout main
+
+popd
+
+unset TAG_TIME
+unset GIT_AUTHOR_DATE
+unset GIT_COMMITTER_DATE
+unset GIT_AUTHOR_NAME
+unset GIT_AUTHOR_EMAIL
+unset GIT_COMMITTER_NAME
+unset GIT_COMMITTER_EMAIL
+
+echo "`cat <<EOF-
+
+The Git repo generated by this script is intended to be published on
```

Comment:
suggestion: can we please move into example-get-started-cv/README.md .. and replace it here with `cat README.md` or "echo Read README.md to ...."

I'm planning to do this in the other repos soon, since it's getting out of hands a bit and we need more info in this section (e.g. mention that some manual stuff is neede)